configuration/analyzer_config.json

```diff
@@ -258,6 +258,34 @@
     "secrets": {},
     "params": {}
   },
+  "CapeSandbox" : {
+    "type" : "file",
+    "python_module" : "cape_sandbox.CAPEsandbox",
+    "description" : "Automatic scan of suspicious files using CapeSandbox API",
+    "disabled" : false,
+    "external_service" : false,
```

Comment:
even if this could be self-managed, I would set it `True` by default

Reply:
Noted